### PR TITLE
Devrel rework - add example notebook

### PR DESCRIPTION
**Pull request by andrewhong5297**

### CODE REVIEW

File: examples/Protocol Wallet Clustering - Analyzing all Artblocks Collectors.ipynb
Comment (bh2smith) [reply examples/Protocol Wallet Clustering - Analyzing all Artblocks Collectors.ipynb:1]: +1

File: README.md
Comment (msf) [inline README.md:64]:
```suggestion
query_results = dune.get_latest_result_dataframe(1215383)
```

**Review by msf — APPROVED** [review]
can you update the jupyter notebook to use `get_latest_result_dataframe()` where appropriate.
 :-) 

thanks, approving it so you don't have to wait for another review

File: README.md
Comment (bh2smith) [reply README.md:64]: This suggested change breaks the context of the readme. Specifically demonstrating the use of max age.

Comment (bh2smith) [reply README.md:64]: That's fine, but then the readme should probably be changed to remove the max age example.

**Comment by msf** [comment]
closing this PR, it is stale by now

**Pull request closed without merging**
